src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-block: 32px 16px;
+  padding-inline: 16px;
```

Comment:
According to the code style rules, shorthand property for padding should be used instead of separate padding-block and padding-inline properties unless you have a specific reason to use them. Consider using `padding: 32px 16px;` for a cleaner and more traditional approach.